endroid/data-sanitize/1.3/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "bundles": {
+        "Endroid\\DataSanitize\\Bundle\\DataSanitizeBundle\\EndroidDataSanitizeBundle": ["all"],
+        "Endroid\\DataSanitize\\Bundle\\DataSanitizeDemoBundle\\EndroidDataSanitizeBundle": ["dev"]
```

Comment:
a demo bundle should not be registered automatically in projects wanting to use the feature.